### PR TITLE
Create Programming Paradigms.md

### PR DESCRIPTION
**Pull request by deepali2002-max**
<hr>

## Description 📜

I have added documentation of various programming paradigms
Fixes # <2424> 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Documentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Review by apeksha235 — CHANGES_REQUESTED** [review]
@deepali2002-max Add it in the folders provided under C_CPP

**Comment by deepali2002-max** [comment]
> @deepali2002-max Add it in the folders provided under C_CPP

Is it okay now?

**Review by deeqakkk — APPROVED** [review]

File: C_CPP/Classes & Objects Programming/readme.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 - [Classes & Objects](Classes&Objects.md)
-- Classes & Objects
+- [Programming Paradigms](https://github.com/deepali2002-max/winter-of-contributing/blob/C_CPP/C_CPP/Classes%20%26%20Objects%20Programming/Programming%20Paradigms.md)
```
Comment (Saviour1001) [inline C_CPP/Classes & Objects Programming/readme.md:2]: the link here references your repository. Add a relative link. [Here](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file) is how.

Comment (deepali2002-max) [reply C_CPP/Classes & Objects Programming/readme.md]: Could you check it now?

**Pull request merged**